Makefile

```diff
@@ -6,6 +6,10 @@
 # - CIRCLECI_TOKEN
 #
 
+AWS_ACCESS_KEY_ID := $(shell aws secretsmanager get-secret-value --secret-id=aws_access_key_id | jq -r '.SecretString')
+AWS_SECRET_ACCESS_KEY := $(shell aws secretsmanager get-secret-value --secret-id=aws_secret_access_key | jq -r '.SecretString')
+CIRCLECI_TOKEN := $(shell aws secretsmanager get-secret-value --secret-id=circleci_api_key | jq -r '.SecretString')
```

Comment:
I'm not sure if these secret-id's are correct.  I wasn't able to get zero to populate secretsmanager.  For local testing, I created the secrets manually.

Reply:
Hmm yeah we'll have to figure this out later. To prevent collisions when re-running, @davidcheung added a timestamp to the secret name. This is probably not a real-world scenario so one option is to remove the timestamp. Otherwise this would be a case where we need to allow the eks stack repo to export it's output and use that as the input for this job, or change from a timestamp to something deterministic, so the `apply` step of each project can make sure they use the same secret name.